event-pubsub.js

```diff
@@ -13,23 +13,22 @@ function sub(type,handler){
 }
 
 function unsub(type,handler){
-    if(!handler){
+    if(!handler && type !== '*'){
         var err=new ReferenceError('handler not defined. if you wish to remove all handlers from the event please pass "*" as the handler');
```

Comment:
The reason not to do this is because if someone passed a reference to a handler out of scope, it will pass as undefined.

This code would allow bad scoping in a developers code to accidentally remove ALL handlers.

We found this out through a few bug requests which were just mistakes on the devs end referencing variables that were not in the scope they thought.